src/otx/algo/utils/mmengine_utils.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Load Checkpoint functions."""
+# TODO(someone): Revisit mypy errors after deprecation of mmlab
+# mypy: ignore-errors
```

Comment:
What kind of errors cannot be handled in this PR?

Reply:
This is copy code of some functions provided by mmengine, so it's not task-specific, and we need to clean it up later, and I think mypy will be worked on at that time.